x-pack/plugins/security_solution/public/common/hooks/timeline/use_query_timeline_by_id_on_url_change.ts

```diff
@@ -75,9 +80,18 @@ export const useQueryTimelineByIdOnUrlChange = () => {
         updateIsLoading: (status: { id: string; isLoading: boolean }) =>
           dispatch(timelineActions.updateIsLoading(status)),
         updateTimeline: dispatchUpdateTimeline(dispatch),
+        unifiedComponentsInTimelineEnabled,
```

Comment:
Thanks for keeping these all constant. Going to make it much easier to remove when this becomes the default